src/InsertionSort.java

```diff
@@ -0,0 +1,14 @@
+public class InsertionSort {
+    public void insertionSort(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            int key = array[i];
```

Comment:
It's not a `key`, it is a pointer to the current element. Rename `key` to `current`